### PR TITLE
Docs/PR code review

### PR DESCRIPTION
**Pull request by mts299**
Here is the last PR documentation on how to give Code Reviews and guidelines to follow #93 

Again looking for suggestions, tips, and if this makes sense to a non-github user.

### CODE REVIEW

File: docs/dev/code_review.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Code Reviews 
+
+Reviewing code is like proof reading a paper, it essential to ensure the followig:
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:3]:
```suggestion
Reviewing code is like proof reading a paper, it is essential to ensure the following: 
```

File: docs/dev/code_review.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Code Reviews 
+
+Reviewing code is like proof reading a paper, it essential to ensure the followig: 
+
+- code is user readable
+- follows consistency
+- helps create collaboration
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:7]:
```suggestion
- helps encourage collaboration
```
?? maybe

File: docs/dev/code_review.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+- structure is proper
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:8]:
```suggestion
- code has a proper structure
```
? I'm not sure what this point means

File: docs/dev/code_review.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
+# Code Reviews 
+
+Reviewing code is like proof reading a paper, it essential to ensure the followig: 
+
+- code is user readable
+- follows consistency
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:6]:
```suggestion
- enforces consistency
```
?? sorry ignore these suggestions if they're a bit picky!

File: docs/dev/code_review.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+- helps create collaboration
+- structure is proper
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
+
+Anyone can code review even if you don't program in the native language, which is python for pyDARN. 
+Doing code reviews can also be good learning experiences to better understand the software and how to program.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:14]:
```suggestion
Code reviewing is also an excellent learning experience which helps you better understand the software and how to program in general. 
```

File: docs/dev/code_review.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+- code is user readable
+- follows consistency
+- helps create collaboration
+- structure is proper
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
+
+Anyone can code review even if you don't program in the native language, which is python for pyDARN.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:13]:
```suggestion
Anyone can code review even if you don't program in the native language, which for pyDARN is python. 
```

File: docs/dev/code_review.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+- structure is proper
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
+
+Anyone can code review even if you don't program in the native language, which is python for pyDARN. 
+Doing code reviews can also be good learning experiences to better understand the software and how to program. 
+
+Code reviews should not take more than an hour and Pull Requests should be less than 400 lines.
+However, this is challenging for new features and major changes.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:17]:
```suggestion
However, these restrictions can be challenging for new features and major changes. 
```

File: docs/dev/code_review.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
+
+Anyone can code review even if you don't program in the native language, which is python for pyDARN. 
+Doing code reviews can also be good learning experiences to better understand the software and how to program. 
+
+Code reviews should not take more than an hour and Pull Requests should be less than 400 lines.
+However, this is challenging for new features and major changes. 
+If this is the case, then code review parts or specific files to reduce your time spent.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:18]:
```suggestion
If this is the case, then you can code review parts of a file or specific files to reduce your time spent. 
```

Comment (carleyjmartin) [reply docs/dev/code_review.md:18]: Could add a 'Every little helps!' on the end?
```suggestion
If this is the case, then you can code review parts of a file or specific files to reduce your time spent. Every little helps!
```
Two suggestions so its easier than you having to open the file and change :P

File: docs/dev/code_review.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+Doing code reviews can also be good learning experiences to better understand the software and how to program. 
+
+Code reviews should not take more than an hour and Pull Requests should be less than 400 lines.
+However, this is challenging for new features and major changes. 
+If this is the case, then code review parts or specific files to reduce your time spent. 
+
+## How to start a Code Review 
+
+1. Pick a Pull Request to code review, which can be found [here](https://github.com/SuperDARN/pydarn/pulls)
+2. Click on `files chaned` at the top of the PR conversation below the title on the right side
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:23]:
```suggestion
2. Click on `files changed` at the top of the PR conversation below the title on the right side 
```

File: docs/dev/code_review.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+Code reviews should not take more than an hour and Pull Requests should be less than 400 lines.
+However, this is challenging for new features and major changes. 
+If this is the case, then code review parts or specific files to reduce your time spent. 
+
+## How to start a Code Review 
+
+1. Pick a Pull Request to code review, which can be found [here](https://github.com/SuperDARN/pydarn/pulls)
+2. Click on `files chaned` at the top of the PR conversation below the title on the right side 
+3. Now review the changes that occurred in this PR - avoid commenting on other lines as this is scope creep.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:24]:
```suggestion
3. Now review the changes that occurred in this PR - avoid commenting on other lines not included in the changes as this is `scope creep`.
```

File: docs/dev/code_review.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+4. If you want to make a comment hover the cursor over the line numbers on the left side and a **+** button will appear. Click it!
+5. Now right any feedback or questions in the box and hit `Start a review` on the bottom right in the pop up window.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:26]:
```suggestion
5. Now write any feedback or questions in the box and hit `Start a review` on the bottom right in the pop up window. 
```

File: docs/dev/code_review.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+1. Pick a Pull Request to code review, which can be found [here](https://github.com/SuperDARN/pydarn/pulls)
+2. Click on `files chaned` at the top of the PR conversation below the title on the right side 
+3. Now review the changes that occurred in this PR - avoid commenting on other lines as this is scope creep.
+4. If you want to make a comment hover the cursor over the line numbers on the left side and a **+** button will appear. Click it!
+5. Now right any feedback or questions in the box and hit `Start a review` on the bottom right in the pop up window. 
+    
+    !!! Warning
+        Do not make several single comments as this can cause a lot of email notifications
+
+6. If you want to change a line in the code use [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/)
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:31]:
```suggestion
6. If you want to change a line in the code use a [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/)
```

File: docs/dev/code_review.md [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+7. If you wish comment on several consecutive lines hold **+** down and drag the cursor down to select multiple lines
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:32]:
```suggestion
7. If you wish to comment on several consecutive lines hold **+** down and drag the cursor down to select multiple lines
```

File: docs/dev/code_review.md [hunk 14]
```diff
@@ -0,0 +1,121 @@
...
+8. Once you are done, click on `Finish your changes` or `Review channges` (if no comments made) then comment on any general feedback in the text box
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:33]:
```suggestion
8. Once you are done, click on `Finish your changes` or `Review changes` (if no comments made) then comment on any general feedback in the text box
```

File: docs/dev/code_review.md [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+9. Select:  
+  - Comment: general comments to be fixed up but nothing major
+  - Approve: everything looks good no changes needed
+  - Request Changes: major changes to make
+      
+    !!! Warning
+        Request changes, present anyone from merging the code. Generally you will need to approve the changes that were addressed. 
+        Generally if you cannot do this then do not use Request changes.
+
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:42]:
```suggestion
    !!! Warning
        Request changes, prevents anyone from merging the code. Generally you will need to approve the changes that were addressed. 
        Generally if you cannot do this then do not use Request changes.

```

File: docs/dev/code_review.md [hunk 16]
```diff
@@ -0,0 +1,121 @@
...
+10. Submit! 
+
+!!! Note
+    If you want to make minor changes directly in the code like typos and grammar changes you can pull the code down described in [testing](testing.md)
+    and change the code as you want and then re-push the code back to the code. The Developer can review the changes in your commit.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:47]:
```suggestion
    and change the code as you want and then re-push the code back to the branch. The Developer can review the changes in your commit. 
```
I'm not sure what this note is trying to say, is it saying how to make changes if you have to go into the files again?

File: docs/dev/code_review.md [hunk 17]
```diff
@@ -0,0 +1,121 @@
...
+    and change the code as you want and then re-push the code back to the code. The Developer can review the changes in your commit. 
+
+### What to look for?
+
+Here is a list of what to look for 
+
+- Does it follow [PEP8](https://www.python.org/dev/peps/pep-0008/) style?
+- Is the copyright and license disclaimer in the file?
+- Did they add the modification line if they made updates?
+- Are there doc strings for the class and or functions?
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:56]:
```suggestion
- Are there doc strings for the class and/or functions?
```

```suggestion
- Are there doc strings for the class and for functions?
```

File: docs/dev/code_review.md [hunk 18]
```diff
@@ -0,0 +1,121 @@
...
+- Are there confusing variable, function, and/or class names?
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:57]:
```suggestion
- Are there confusing variables, functions, and/or class names? 
```

File: docs/dev/code_review.md [hunk 19]
```diff
@@ -0,0 +1,121 @@
...
+- Did they add the modification line if they made updates?
+- Are there doc strings for the class and or functions?
+- Are there confusing variable, function, and/or class names? 
+- Are there defaults and do they make sense for the function? 
+- Is the code too complex? Could it be simpler? 
+- Do the error messages/exceptions make sense and are helpful?
+- Could there be more comments on certain sections?
+- Are hard coded values documented? 
+- Are algorithms or mathematical equations cited to publication, webpage, or book? 
+- Nomenclature make sense?
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:64]:
```suggestion
- Does the nomenclature make sense? 
```

File: docs/dev/code_review.md [hunk 20]
```diff
@@ -0,0 +1,121 @@
...
+- Are there doc strings for the class and or functions?
+- Are there confusing variable, function, and/or class names? 
+- Are there defaults and do they make sense for the function? 
+- Is the code too complex? Could it be simpler? 
+- Do the error messages/exceptions make sense and are helpful?
+- Could there be more comments on certain sections?
+- Are hard coded values documented? 
+- Are algorithms or mathematical equations cited to publication, webpage, or book? 
+- Nomenclature make sense? 
+- Are there any cases they may need to consider?
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:65]:
```suggestion
- Are there any cases they may need to reconsider? 
```
I think? I'm not sure what this is supposed to say.

File: docs/dev/code_review.md [hunk 21]
```diff
@@ -0,0 +1,121 @@
...
+
+## How to speed up code reviews
+
+Here are some tips and tricks:
+
+- Dedicated a certain amount of time and just review what you can in that time period 
+- Scan missing doc string and obvious mistakes 
+- Focus on what has been changed 
+- If the issue repeats then make a comment for them to fix the rest and add in the general feedback when submitting the comments 
+- Do not get too pedantic!
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:76]: Good luck with this one :P

File: docs/dev/code_review.md [hunk 22]
```diff
@@ -0,0 +1,121 @@
...
+- Break up the reviews if you can only review a few files or functions then do it and mention it. 
+
+## Reviewing a review 
+  
+As a developer you will need to address any comments that come up from code reviews like you would with reviewers comments in paper submission.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:81]:
```suggestion
As a developer you will need to address any comments that come up from code reviews like you would with reviewers comments in a paper submission. 
```

File: docs/dev/code_review.md [hunk 23]
```diff
@@ -0,0 +1,121 @@
...
+then at the top in the middle there is `commit suggestions` and this commit all the suggestions for you. 
+
+### Responding to Comments 
+
+If you don't agree with a suggestion try to find a compromise or acknowledge their effort and explain why you prefer your style.
+Remember to hit `Start a review` to prevent multiple email notifications. 
+
+If you made the change requested, then select `resolve conflict` so they know you addressed their comment.
+
+If you are answer question or responding to some feedback remember to acknowledge their time, then address it in collaborative manner.
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:98]:
```suggestion
If you are answering a question or responding to some feedback, remember to acknowledge their time and effort, then address it in collaborative manner. 
```

File: docs/dev/code_review.md [hunk 24]
```diff
@@ -0,0 +1,121 @@
...
+- Dedicate time to review of a given period, avoid being distracted 
+- Ask questions instead of stating 
+- Remember to comment on the code 
+- Acknowledge time spent and saying "Thank you" 
+- Suggest solutions or help
+- Avoid aggressive terminology 
+- Avoid reviewing in a bad mood or feeling pressured/rush 
+- Review all comments to make sure it sounds helpful and not demeaning 
+- Avoid pedantic points, focus on what is needed and not personal preferences 
+- Any amount of reviewing small or large is welcoming!
```
Comment (carleyjmartin) [inline docs/dev/code_review.md:116]:
```suggestion
- Any amount of reviewing small or large is welcome! 
```

**Review by carleyjmartin — COMMENTED** [review]
This appears like it's a lot, but it's 99% spelling or words to be added, if I'm not sure I've tried to give a few suggestions to make your life easier!

File: docs/dev/code_review.md (hunk 16 above)
Comment (mts299) [reply docs/dev/code_review.md:47]: @Devin-Huyghebaert suggested it like you could have checked out this branch and made changes within the file?

File: docs/dev/code_review.md (hunk 21 above)
Comment (mts299) [reply docs/dev/code_review.md:76]: Yea....

Comment (mts299) [reply docs/dev/code_review.md:76]: Wishful  thinking

**Review by carleyjmartin — APPROVED** [review]
That was absolutely rapid :P

File: docs/dev/code_review.md (hunk 16 above)
Comment (carleyjmartin) [reply docs/dev/code_review.md:47]: Ah okay, so code is good!

File: docs/dev/code_review.md [hunk 25]
```diff
@@ -0,0 +1,121 @@
...
+3. Now review the changes that occurred in this PR - avoid commenting on other lines not included in the changes as this is `scope creep`.
+4. If you want to make a comment hover the cursor over the line numbers on the left side and a **+** button will appear. Click it!
+5. Now write any feedback or questions in the box and hit `Start a review` on the bottom right in the pop up window. 
+    
+    !!! Warning
+        Do not make several single comments as this can cause a lot of email notifications
+
+6. If you want to change a line in the code use a [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/)
+7. If you wish to comment on several consecutive lines hold **+** down and drag the cursor down to select multiple lines
+8. Once you are done, click on `Finish your changes` or `Review changes` (if no comments made) then comment on any general feedback in the text box
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:33]: Here is where I would suggest potentially including what type of review was done, a Formatting Review and/or a Testing Review.

File: docs/dev/code_review.md [hunk 26]
```diff
@@ -0,0 +1,121 @@
...
+9. Select:  
+  - Comment: general comments to be fixed up but nothing major
+  - Approve: everything looks good no changes needed
+  - Request Changes: major changes to make
+
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:38]: I am guessing that if only a partial review is completed for the submitted changes, comment should be selected.  If a future review is submitted that advances the previous review, should this be mentioned in the later review?  Is it possible to refer to a previous review/comment?

File: docs/dev/code_review.md [hunk 27]
```diff
@@ -0,0 +1,121 @@
...
+- Avoid aggressive terminology 
+- Avoid reviewing in a bad mood or feeling pressured/rush 
+- Review all comments to make sure it sounds helpful and not demeaning 
+- Avoid pedantic points, focus on what is needed and not personal preferences 
+- Any amount of reviewing small or large is welcome! 
+- Approve! If things look good approve the PR! Approvals take us one step closer to being able to merge the code!
+
+!!! Note
+    If you approve the PR and you are the second to approve, or there are already 2 approvals and all concerns are addressed 
+    please merge the PR! Unless it is a release PR or a PR specifically looking for more than two approvals.
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:121]: As mentioned previously, perhaps a clarification on the different types of reviews should be discussed.  Two approvals of the format/syntax of the code, with 0 approvals on the testing of the code is not ideal.

File: docs/dev/code_review.md [hunk 28]
```diff
@@ -0,0 +1,121 @@
...
+- Ask questions instead of stating 
+- Remember to comment on the code 
+- Acknowledge time spent and say "Thank you" 
+- Suggest solutions or help
+- Avoid aggressive terminology 
+- Avoid reviewing in a bad mood or feeling pressured/rush 
+- Review all comments to make sure it sounds helpful and not demeaning 
+- Avoid pedantic points, focus on what is needed and not personal preferences 
+- Any amount of reviewing small or large is welcome! 
+- Approve! If things look good approve the PR! Approvals take us one step closer to being able to merge the code!
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:117]:
```suggestion
- Approve! If things look good approve the PR! Approvals take us one step closer to being able to merge the code!  Be sure to include in the approval comment what was tested and is being approved.
```

File: docs/dev/code_review.md [hunk 29]
```diff
@@ -0,0 +1,121 @@
...
+
+Reviewing code is like proof reading a paper, it is essential to ensure the following: 
+
+- code is user readable
+- enforces consistency
+- helps encourage collaboration
+- code has a proper structure
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
```
[the remark this answers is not in the document]
Comment (mts299) [reply docs/dev/code_review.md]: Take a look at my changes and resolve the convo if it looks good.

File: docs/dev/code_review.md (hunk 27 above)
Comment (mts299) [reply docs/dev/code_review.md]: Take a look at my changes and resolve the convo if it looks good.

**Review by mts299 — COMMENTED** [review]
I have made changes @Devin-Huyghebaert based on our slack chat and your comments even added another note on the bottom to let people reach out if things are scary. Let me know what you think!

File: docs/dev/code_review.md [hunk 30]
```diff
@@ -0,0 +1,143 @@
+# Code Reviews 
+
+Reviewing code is like proof reading a paper, it is essential to ensure the following: 
+
+- code is user readable
+- enforces consistency
+- helps encourage collaboration
+- code has a proper structure
+- Reviewing/Verifying algorithms
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:9]:
```suggestion
- The algorithms and software are tested and verified 
```

File: docs/dev/code_review.md [hunk 31]
```diff
@@ -0,0 +1,143 @@
...
+3. Now review the changes that occurred in this PR - avoid commenting on other lines not included in the changes as this is `scope creep`.
+4. If you want to make a comment hover the cursor over the line numbers on the left side and a **+** button will appear. Click it!
+5. Now write any feedback or questions in the box and hit `Start a review` on the bottom right in the pop up window. 
+    
+    !!! Warning
+        Do not make several single comments as this can cause a lot of email notifications
+
+6. If you want to change a line in the code use a [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/)
+7. If you wish to comment on several consecutive lines hold **+** down and drag the cursor down to select multiple lines
+8. Once you are done, click on `Finish your changes` or `Review changes` (if no comments made) then comment on any general feedback in the text box. This should include what type of review you did:
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:39]:
```suggestion
8. Once you are done, click on `Finish your changes` or `Review changes` (if no comments made) then comment on any general feedback in the text box. This should include what type of review you did and if you will be continuing your review at a later time/date.  Some examples include:
```

File: docs/dev/code_review.md [hunk 32]
```diff
@@ -0,0 +1,143 @@
...
+  - Testing and a bit of code review to find the bug
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:40]:
```suggestion
  - Testing and code review to find a bug 
```

File: docs/dev/code_review.md [hunk 33]
```diff
@@ -0,0 +1,143 @@
...
+    !!! Warning
+        Do not make several single comments as this can cause a lot of email notifications
+
+6. If you want to change a line in the code use a [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/)
+7. If you wish to comment on several consecutive lines hold **+** down and drag the cursor down to select multiple lines
+8. Once you are done, click on `Finish your changes` or `Review changes` (if no comments made) then comment on any general feedback in the text box. This should include what type of review you did:
+  - Testing and a bit of code review to find the bug 
+  - Code review on style/formatting 
+  - Partial code review 
+  - Report if you will be able to continue reviewing or want to test the code as well.
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:43]:
```suggestion
```

File: docs/dev/code_review.md [hunk 34]
```diff
@@ -0,0 +1,143 @@
...
+- Avoid aggressive terminology 
+- Avoid reviewing in a bad mood or feeling pressured/rush 
+- Review all comments to make sure it sounds helpful and not demeaning 
+- Avoid pedantic points, focus on what is needed and not personal preferences 
+- Any amount of reviewing small or large is welcome! 
+- Approve! Make sure to comment what type of approval you are giving, complete testing or a complete reviewing of the code. This will help others know what needs to be done to get the PR closer to `merge` 
+
+!!! Note
+    If you approve the PR and you are the second to approve, or there are already 2 approvals and all concerns are addressed 
+    and one approval is testing and one is code review, please merge the PR! Unless it is a release PR or a PR specifically looking for more than two approvals.
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:140]:
```suggestion
    with one approval on testing and one approval on code review, please merge the PR! An exception to this is if  it is a release PR or if more than two approvals are requested in the PR instructions. 
```

File: docs/dev/code_review.md [hunk 35]
```diff
@@ -0,0 +1,142 @@
...
+9. Select:  
+  - Comment: general comments to be fixed up but nothing major
+    
+    !!! Note
+        If you did a partial review please select comment. Approve is only done for complete reviews or complete testing 
+
+  - Approve: everything looks good no changes needed
+    
+    !!! Note
+        This should only be done on complete code reviews. You can also, revert your approval after it is made if you find something else later.
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:52]:
```suggestion
        This should only be done on complete code reviews. You can also revert your approval after it is made if you find something else later. 
```

File: docs/dev/code_review.md [hunk 36]
```diff
@@ -0,0 +1,142 @@
...
+    
+    !!! Note
+        This should only be done on complete code reviews. You can also, revert your approval after it is made if you find something else later. 
+        Do this by going down to the merge button and find your name under approvals and click the three dots and select `re-review` 
+  
+  - Request Changes: major changes to make
+      
+    !!! Warning
+        Request changes, prevents anyone from merging the code. Generally you will need to approve the changes that were addressed. 
+        Generally if you cannot do this then do not use Request changes.
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:59]:
```suggestion
        If you cannot do this then do not use Request changes.
```

File: docs/dev/code_review.md [hunk 37]
```diff
@@ -0,0 +1,142 @@
...
+- Are algorithms or mathematical equations cited to publication, webpage, or book? 
+- Does the nomenclature make sense? 
+- Are there any cases they may need to reconsider? 
+- Would comments or documentation make sense to the general user?
+
+## How to speed up code reviews
+
+Here are some tips and tricks:
+
+- Dedicated a certain amount of time and just review what you can in that time period
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:90]:
```suggestion
- Dedicate a certain amount of time and just review what you can in that time period 
```

File: docs/dev/code_review.md [hunk 38]
```diff
@@ -0,0 +1,142 @@
...
+- Avoid aggressive terminology 
+- Avoid reviewing in a bad mood or feeling pressured/rush 
+- Review all comments to make sure it sounds helpful and not demeaning 
+- Avoid pedantic points, focus on what is needed and not personal preferences 
+- Any amount of reviewing small or large is welcome! 
+- Approve! Make sure to comment what type of approval you are giving, complete testing or a complete reviewing of the code. This will help others know what needs to be done to get the PR closer to `merge` 
+
+!!! Note
+    If you approve the PR and you are the second to approve, or there are already 2 approvals and all concerns are addressed 
+    with one approval on testing and one approval on code review, please merge the PR! An exception to this is if  it is a release PR or if more than two approvals are requested in the PR instructions.
```
Comment (Devin-Huyghebaert) [inline docs/dev/code_review.md:139]:
```suggestion
    with one approval on testing and one approval on code review, please merge the PR! An exception to this is if it is a release PR or if more than two approvals are requested in the PR instructions. 
```

**Review by Devin-Huyghebaert — APPROVED** [review]

**Comment by mts299** [comment]
@Devin-Huyghebaert or @carleyjmartin wanna do the merge?

**Comment by Devin-Huyghebaert** [comment]
> @Devin-Huyghebaert or @carleyjmartin wanna do the merge?

I will let @carleyjmartin merge if she agrees to the changes added including the 2 different types of approvals: testing approval and format/style approval.

**Pull request merged**